src/Controls/src/Core/Entry/Entry.Android.cs

```diff
@@ -30,5 +30,18 @@ public static void MapText(IEntryHandler handler, Entry entry)
 		{
 			Platform.EditTextExtensions.UpdateText(handler.PlatformView, entry);
 		}
+
+		static void MapFocus(IViewHandler handler, IView view, object args)
+		{
+			if (view is not VisualElement ve)
+				return;
+
+			if (ve.IsFocused)
+			{
+				KeyboardManager.ShowKeyboard((handler as IPlatformViewHandler).PlatformView);
```

Comment:
The as casting can return an null, that means that '.PlatformView' can throw a NRE, but be careful, because we CANNOT do:
```csharp
KeyboardManager.ShowKeyboard((handler as IPlatformViewHandler)?.PlatformView);
```
The reason for that is that the ShowKeyboard method is implemented as follows:
```csharp
internal static void ShowKeyboard(this TextView inputView)
                {
                        if (inputView?.Context is null)
                                throw new ArgumentNullException(nameof(inputView) + " must be set before the keyboard can be shown.");

                        using (var inputMethodManager = inputView.Context.GetSystemService(Context.InputMethodService) as InputMethodManager)
                        {
                                // The zero value for the second parameter comes from
                                // https://developer.android.com/reference/android/view/inputmethod/InputMethodManager#showSoftInput(android.view.View,%20int)
                                // Apparently there's no named value for zero in this case
                                inputMethodManager?.ShowSoftInput(inputView, 0);
                        }
                }

                internal static void ShowKeyboard(this SearchView searchView)
                {
                        if (searchView?.Context is null || searchView?.Resources is null)
                        {
                                throw new ArgumentNullException(nameof(searchView));
                        }

                        var queryEditor = searchView.GetFirstChildOfType<EditText>();

                        if (queryEditor is null)
                                return;

                        using (var inputMethodManager = (InputMethodManager)searchView.Context.GetSystemService(Context.InputMethodService)!)
                        {
                                // The zero value for the second parameter comes from
                                // https://developer.android.com/reference/android/view/inputmethod/InputMethodManager#showSoftInput(android.view.View,%20int)
                                // Apparently there's no named value for zero in this case
                                inputMethodManager?.ShowSoftInput(queryEditor, 0);
                        }
                }

                internal static void ShowKeyboard(this AView view)
                {
                        switch (view)
                        {
                                case SearchView searchView:
                                        searchView.ShowKeyboard();
                                        break;
                                case TextView textView:
                                        textView.ShowKeyboard();
                                        break;
                        }
                }
```
All the implementation will throw a NRE if we pass null as the platform view, therefore, we probably want to do the cast using the 'is' operator inside the if statement.

Reply:
Updated, thanks - let me know if my change captures what you're describing here or if I missed something!